malariagen_data/anopheles.py

```diff
@@ -4406,6 +4412,105 @@ def plot_ihs_gwss_track(
+        window_size: xpehh_params.window_size = xpehh_params.window_size_default,
+        percentiles: xpehh_params.percentiles = xpehh_params.percentiles_default,
+        standardize: xpehh_params.standardize = True,
+        standardization_bins: Optional[xpehh_params.standardization_bins] = None,
+        standardization_n_bins: xpehh_params.standardization_n_bins = xpehh_params.standardization_n_bins_default,
+        standardization_diagnostics: xpehh_params.standardization_diagnostics = False,
+        filter_min_maf: xpehh_params.filter_min_maf = xpehh_params.filter_min_maf_default,
+        map_pos: Optional[xpehh_params.map_pos] = None,
+        min_ehh: xpehh_params.min_ehh = xpehh_params.min_ehh_default,
+        max_gap: xpehh_params.max_gap = xpehh_params.max_gap_default,
+        gap_scale: xpehh_params.gap_scale = xpehh_params.gap_scale_default,
+        include_edges: xpehh_params.include_edges = True,
+        use_threads: xpehh_params.use_threads = True,
+        min_cohort_size: Optional[
+            base_params.min_cohort_size
+        ] = xpehh_params.min_cohort_size_default,
+        max_cohort_size: Optional[
+            base_params.max_cohort_size
+        ] = xpehh_params.max_cohort_size_default,
+        random_seed: base_params.random_seed = 42,
+        palette: xpehh_params.palette = xpehh_params.palette_default,
+        title: Optional[gplt_params.title] = None,
+        sizing_mode: gplt_params.sizing_mode = gplt_params.sizing_mode_default,
+        width: gplt_params.width = gplt_params.width_default,
+        track_height: gplt_params.track_height = 170,
+        genes_height: gplt_params.genes_height = gplt_params.genes_height_default,
+        show: gplt_params.show = True,
+        output_backend: gplt_params.output_backend = gplt_params.output_backend_default,
+    ) -> gplt_params.figure:
+        # gwss track
+        fig1 = self.plot_xpehh_gwss_track(
+            contig=contig,
+            analysis=analysis,
+            sample_sets=sample_sets,
+            cohort1_query=cohort1_query,
+            cohort2_query=cohort2_query,
+            window_size=window_size,
+            percentiles=percentiles,
+            palette=palette,
+            standardize=standardize,
+            standardization_bins=standardization_bins,
+            standardization_n_bins=standardization_n_bins,
+            standardization_diagnostics=standardization_diagnostics,
+            filter_min_maf=filter_min_maf,
+            map_pos=map_pos,
+            min_ehh=min_ehh,
+            max_gap=max_gap,
+            gap_scale=gap_scale,
+            include_edges=include_edges,
+            use_threads=use_threads,
+            min_cohort_size=min_cohort_size,
+            max_cohort_size=max_cohort_size,
+            random_seed=random_seed,
+            title=title,
+            sizing_mode=sizing_mode,
+            width=width,
+            height=track_height,
+            show=False,
+            x_range=None,
```

Comment:
```suggestion
            x_range=None,
            output_backend=output_backend,
```

Need to pass this parameter through so that all subfigures use webgl.